src/js/components/container/ButtonContainer.js

```diff
@@ -1,22 +1,53 @@
 import React, { Component } from "react";
 import Button from "../presentational/Button";
 import ButtonGroup from "../presentational/styled/ButtonGroup";
+import { CompactPicker } from 'react-color'
+
+//Must run: npm install react-color --save
```

Comment:
Thanks for the tip! Because you've done work of adding it to the package.json all we need to do is run `npm install`.

Reply:
Oh ok, good to know!